starters/deno-oak-denodb/src/cache/Cache.ts

```diff
@@ -0,0 +1,157 @@
+import {
+	Context,
+	graphql,
+	GraphQLSchema,
+	makeExecutableSchema,
+	Middleware,
+	Redis,
+	renderPlaygroundPage,
+	Router,
+	SetOpts,
+} from '../../deps.ts';
+
+type GetFreshValue<Value> = {
+	(): Promise<Value> | Value;
+};
+
+type ArgsType = {
+	schema: {
+		typeDefs: any;
+		resolvers: any;
+	};
+	redisClient: Redis;
+	usePlayground: boolean;
+	route: string;
+	defaultEx?: number | undefined;
+};
+
+export class Cache {
```

Comment:
It seems like this class is doing a bit too much. By the name of it, it should only be a wrapper around the Redis library, but here it's also handling GraphQL requests.

I would suggest to have this class perform the functionality of the "Cache API Wrapper" from the [specification](https://docs.google.com/document/d/1iEXK6BFgu5HZ4dfU7pfv80zLOAd86T8AzstCMEYTUXA/edit#), where its public interface only has the following methods from the spec:

- Read item
- Write item
- Invalidate item

Also, the constructor can probably be without arguments. It can instantiate a new Redis instance internally.